release/terraform/main.tf

```
@@ -0,0 +1,52 @@
+terraform {
+  backend "gcs" {
+    prefix  = "terraform/state"
+  }
+}
+
+provider "google" {
+  project = var.project
+}
+
+data "local_file" "regions_file" {
+  filename = "region_to_dataset_suffix_map.yaml"
+}
+
+locals {
+  regions_map = yamldecode(data.local_file.regions_file.content)
+}
+
+resource "google_storage_bucket" "regional_bucket" {
+  for_each                    = local.regions_map
+  name                        = "${var.project}-lib-${replace(each.value, "_", "-")}"
+  uniform_bucket_level_access = true
+  location                    = each.key
+  force_destroy               = true
```

```suggestion
  force_destroy               = false
```

Let's not accidentally delete any buckets which have objects